bulk_download.py

```diff
@@ -61,19 +62,40 @@ def log_result(log_path, msg):
 
 if __name__ == '__main__':
 
-    print(Path.cwd())
+    api_choice = input("Aplha Vantage or Yahoo finance? A/Y")
 
-    av_key = input('Insert your Alpha Vantage api key: ')
+    if(api_choice == "A"):
```

Comment:
Well yes we would need to refactor bulk_download.py, it was just meant to be a very specific main script